backend/coins/btc/transactions/transactions_test.go

```diff
@@ -184,7 +184,6 @@ func (s *transactionsSuite) TestUpdateAddressHistorySingleTxReceive() {
 	)
 	transactions := s.transactions.Transactions(func(blockchainpkg.ScriptHashHex) bool { return false })
 	require.Len(s.T(), transactions, 1)
-	require.Equal(s.T(), tx1, transactions[0].Tx)
```

Comment:
So, how do you ensure `transactions[0]` is what's expected? Because the `Height`, balance and utxo match? Just checking.